### PR TITLE
Copyright notice

### LINKED ISSUE #1937: Footer copyright Notice
**Actual Behaviour**

Right, now there's no copyright notice in the footer

**Expected Behaviour**

Should show Copyright along with dynamic date year

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by sweezy793**
Fixes #1937 

Changes: Added  copyright notice with dynamic year date

Surge Deployment Link: https://pr-1938-fossasia-susi-skill-cms.surge.sh

Screenshots for the change: 
ss1

### CODE REVIEW

**Comment by sweezy793** [comment]
> capture
> Please fix this.

@plxity Done! Please check again :)

**Comment by neeraj3029** [comment]
screenshot 2019-02-19 at 3 03 35 pm
Please fix the alignment of SUSI logo with other elements

**Comment by sweezy793** [comment]
@neeraj3029 there are no alignment problems with SUSI logo and other elements on medium to large displays. When transitioning to lower displays (around 900 x 300) a bit of padding is added so as to make space for Copyright message. On most large, medium and mobile displays, it looks fine.

**Comment by neeraj3029** [comment]
@sweezy793 I would suggest you to observe the positioning of logo at width 840 to 960 px, you will find it a bit low.

**Comment by sweezy793** [comment]
@neeraj3029 actually that's because the 2019 in copyright breaks line around that width, so it increases the height of footer, which makes the logo appear a bit low in order for it to align with copyright message.

**Comment by neeraj3029** [comment]
In that case you should just squash the commits :)

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -116,4 +125,9 @@
         width: 100%;
         justify-content: center;
     }
+    .footer-copyright{
+        display: flex;
+        width: 100%;
+        justify-content: center;
+    }
 }
```
Comment (AakashMallik) [inline src/components/Footer/Footer.css]: Add new line here.

**Comment by plxity** [comment]
One more thing, in mobile view order of elements in footer in not correct. Copyright is in the middle. Fix that too.

**Comment by sweezy793** [comment]
> One more thing, in mobile view order of elements in footer in not correct. Copyright is in the middle. Fix that too.

As in desktop view copyright is in the middle, so in mobile view also it's kept in the middle position. If not in middle where else do you suggest ?

**Comment by sweezy793** [comment]
> Don't really wanna nitpick here but if **moment** is being used anywhere in the repo, try using it instead of the date object. The code base would be more consistent that way.

Will change it to Moment then :)

**Comment by sweezy793** [comment]
@AakashMallik Please review the changes :)

**Comment by sweezy793** [comment]
> I think the position of copyright is wrong. It should be on the right side. The existing footer content can be shifted to left side. Thoughts @akshatnitd @anshumanv @AakashMallik

ss2
 Does this look good to you?

**Pull request merged**